modules/ROOT/pages/dataweave-selectors.adoc

```diff
@@ -586,6 +586,86 @@ The script outputs a JSON array with all `name` values.
 ]
 ----
 
+=== Example: Selecting Descending Values
+
+This example selects the descending `Name` values of the `Item` elements.
+
+==== Input Payload
+
+The input payload contains a set of `Name` elements that are nested
+at different levels.
+
+[source,xml, linenums]
+----
+<Example>
+    <Brand>
+        <Id>32345</Id>
+        <logo>circle</logo>
+        <Item>
+            <Name>Perfume</Name>
+            <Item>
+                <Name>Bosque</Name>
+            </Item>
+        </Item>
+        <Item>
+            <Name>t-Shirt</Name>
+        </Item>
+    </Brand>
+    <Brand>
+        <Id>435678C</Id>
+        <logo>circle</logo>
+        <Item>
+            <Name>t-Shirt2</Name>
+            <Item>
+                <Name>t-Shirt red</Name>
+                <Item>
+                    <Name>t-Shirt red with logo</Name>
+                </Item>
+            </Item>
+        </Item>
+    </Brand>
+</Example>
+----
+
+==== DataWeave Source
+
+The DataWeave script uses `..*` to select the values of all `Name` elements nested in the `Item` elements from the input payload and output those values into a XML array.
+
+
+[source,DataWeave, linenums]
+----
+%dw 2.0
+output application/xml
+---
+{
+    Supermarket: {
+        Item: {
+            Value: payload..*Item.Name
+        }
+    }
+}
+----
+
+==== Output
+
+The script outputs a XML array with all `name` values.
+
+[source,json,linenums]
```

Comment:
Quick question here, shouldn't it be xml instead of json?

Reply:
Yes, thanks for catching that. I updated it. Could you reapprove?